packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -79,13 +79,19 @@ async function executeCompatibilityCheck(templateDir, yarnInstall) {
 
     if (
       foundNodeVersionIsLessThanRequired ||
-      foundNodeVersionIsLessThanRequired
+      foundYarnVersionIsLessThanRequired
     ) {
       const errorMessages = [
-        { type: 'node', ok: foundNodeVersionIsLessThanRequired },
-        { type: 'yarn', ok: foundYarnVersionIsLessThanRequired },
```

Comment:
I got the meanings of these values wrong. If one of this is true, it's not ok. So I just renamed the prop to indicate this.